src/Emulator/Tabs/Tabs.tsx

```diff
@@ -19,26 +19,28 @@ const Tabs = () => {
 
   return (
     <div className="tabs">
-      <div className="tabs-container">
-        <div
-          className={activeTab === CONSTANTS.CONFIGURE ? 'tab active' : 'tab'}
-          onClick={() => changeTabHandler(CONSTANTS.CONFIGURE)}
-        >
-          Configure
+      <div className="outer-tabs-container">
+        <div className="tabs-container">
+          <div
+            className={activeTab === CONSTANTS.CONFIGURE ? 'tab active' : 'tab'}
+            onClick={() => changeTabHandler(CONSTANTS.CONFIGURE)}
+          >
+            Configure
+          </div>
+          <div
+            className={activeTab === CONSTANTS.CONSOLE ? 'tab active' : 'tab'}
+            onClick={() => changeTabHandler(CONSTANTS.CONSOLE)}
+          >
+            Console
+          </div>
+          <div
+            className={activeTab === CONSTANTS.HISTORY ? 'tab active' : 'tab'}
+            onClick={() => changeTabHandler(CONSTANTS.HISTORY)}
+          >
+            History
+          </div>
+          <div className="tab empty"></div>
```

Comment:
`<div className="tab empty"></div>` can become `<div className="tab empty" />`
If there is no content within a tag, it can be self closing in JSX. You dont need to change it in this PR but this should be something you're aware of for the future.

Reply:
Yeah, my bad I didn't notice this. I didn't really touch any of the tsx files at all, I was just doing the css on the elements that were already there